bundles/org.openhab.binding.juicenet/src/main/java/org/openhab/binding/juicenet/internal/handler/JuiceNetDeviceHandler.java

```diff
@@ -253,6 +253,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
                     api.setOverride(Objects.requireNonNull(token), energyAtPlugin, overrideTime, energyToAdd);
 
+                    break;
+                }
+                case CHANNEL_OVERRIDE: {
+                    if (command instanceof OnOffType) {
+                        String state = ((OnOffType) command).toString();
+                        if (state.equals("OFF")) {
+                            api.setOverride(Objects.requireNonNull(token), 0, ((long) 0), 0);
+                        }
+                    }
```

Comment:
This can be simplified like that:
```
                    if (command instanceof OnOffType onOffCommand && OnOffType.OFF.equals(onOffCommand)) {
                        api.setOverride(Objects.requireNonNull(token), 0, ((long) 0), 0);
                    }
```

Reply:
Ok, did it like that, looks better I think